store/postgres/migrations/2019-02-01-045927_add_entities_indexes/up.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX IF NOT EXISTS entities_subgraph_entity_idx
+ON entities
+USING btree (subgraph, entity);
```

Comment:
Should we add `id` here at the end? As https://github.com/graphprotocol/graph-node/issues/724#issuecomment-459462836 explains, `(subgraph, entity, id)` covers `(subgraph, entity)` but on top of that also speeds up queries that also include the ID.

Reply:
I think because we already have the pk index covering those three columns, we don't really need to add the `id` in here. I would only add it if we run into query plans that would benefit from it (though I am not sure what they would be right now)